actions/concurrent_runner.go

```diff
@@ -74,6 +78,24 @@ func (r *ConcurrentRunner) Start(a Action, timeout time.Duration) {
 		r.l.Errorf("Ignoring Start: %s.", err)
 		return
 	}
+	actionID, actionType := a.ID(), a.Type()
+
+	var requirements []lock.Entity
+	switch actionType {
+	case "pbm-switch-pitr":
+		requirements = append(requirements, lock.PBM)
+	}
```

Comment:
I suppose this should be moved tho the actions/jobs, so they will be able to provide their requirements. But I prefer to do that in a separate ticket.